src/commands/cmd_stream.cc

```diff
@@ -358,6 +360,101 @@ class CommandXClaim : public Commander {
   }
 };
 
+class CommandAutoClaim : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    CommandParser parser(args, 1);
+    key_name_ = GET_OR_RET(parser.TakeStr());
+    group_name_ = GET_OR_RET(parser.TakeStr());
+    consumer_name_ = GET_OR_RET(parser.TakeStr());
+    {
+      auto s = parser.TakeInt<uint64_t>();
+      if (!s.IsOK()) {
+        return {Status::RedisParseErr, "Invalid min-idle-time argument for XAUTOCLAIM"};
+      }
+      options_.min_idle_time_ms = s.GetValue();
+    }
+    {
```

Comment:
Why? What is the benefit of this additional indentation? If you have this extra block, maybe you need to extract it to a separate function. Otherwise, I think it looks weird and shifts the code to the right for no reason, adding cognitive complexity.

Reply:
Hi @torwig ,

The reason of using a new block is for sharing same variable name with the above scope.
I will change these two variables into to precise names to reduce the redundant block.

Best Regards,
Edward